file_organizer_v2/src/file_organizer/cli/profile.py

```
@@ -0,0 +1,509 @@
+        manager = get_profile_manager()
+        migrator = ProfileMigrator(manager)
+
+        backup = not no_backup
+
+        success = migrator.migrate_version(name, to_version, backup=backup)
+
+        if success:
+            click.echo(f"✓ Migrated profile '{name}' to version {to_version}")
+        else:
+            click.echo("✗ Failed to migrate profile", err=True)
+            raise click.Abort() from e
```

Ruff (0.14.13)

Undefined name `e`

(F821)